app/models/user.rb

```diff
@@ -3,8 +3,12 @@ class User < ApplicationRecord
          :recoverable, :rememberable, :trackable, :validatable
   validates :name, :surname, presence: true,
                              length: { in: 3..30 },
-                             format: { with: /\A[a-zA-Z]+\z/ }
+                             format: { with: /\A[a-zA-Z]+\z/ },
+                             unless: :skip_register_validation
```

Comment:
do you need to skip all of the validations?

Reply:
![screenshot from 2018-04-10 16-32-45](https://user-images.githubusercontent.com/15639895/38606759-98d7147a-3d7f-11e8-87d8-6940977a7df7.png)
Without skipping I had errors for all fields.